lib/galaxy/authnz/__init__.py

```diff
@@ -24,13 +24,13 @@ def __init__(self, provider, config, backend_config):
         :type provider: string
         :param provider: is the name of the identity provider (e.g., Google).
 
-        :type config: xml.etree.ElementTree.Element
+        :type config: lxml.etree.ElementTree._Element
```

Comment:
Just a doc so I'm probably bikeshedding here but does this make sense when the parser might not be lxml?